servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/RedoPublisher.java

```diff
@@ -49,9 +49,9 @@ final class RedoPublisher<T> extends AbstractNoHandleSubscribePublisher<T> {
     void handleSubscribe(Subscriber<? super T> subscriber, ContextMap contextMap,
                          AsyncContextProvider contextProvider) {
         // For the current subscribe operation we want to use contextMap directly, but in the event a re-subscribe
-        // operation occurs we want to restore the original state of the AsyncContext map, so we save a copy upfront.
+        // operation occurs we want to restore the original state of the AsyncContext map.
```

Comment:
Is this comment now still accurate? (original state will not be preserved anymore if a re-subscribe occurs, or?)

Reply:
I clarified these comments.